File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
        public String solution(String[] seoul) {
            String answer = "";
            int f=0;
            for (int i = 0; i < seoul.length; i++) {
                if(seoul[i].equals("Kim")){
                    f=i;
                    break;
                }
            }
            return "김서방은 "+f+"에 있다";
        }
    }

File: 프로그래머스/lv1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.kt
 class Solution {
     fun solution(seoul: Array<String>): String {
         var answer = ""
         for((idx,name) in seoul.withIndex()){
             if(name.equals("Kim")){
                 return "김서방은 ${idx}에 있다"
             }
         }
         return answer
     }
 }